Makefile

```diff
@@ -15,26 +15,26 @@
 #
 
```

Comment:
Adding in a `export GO111MODULE=on` to the Makefile may be advisable, so that even if this this repo is in the GOPATH, go will use the go.mod file.

Reply:
Looks reasonable to me. Can you send a pull request to add that?